src/pages/tokens/[id].tsx

```diff
@@ -141,6 +156,30 @@ function ListLeft ({ token }: { token: TokenData }): JSX.Element {
   )
 }
 
+function BackingAddress ({ tokenName }: {tokenName: string}): JSX.Element {
+  const isEth = tokenName === TokensWithBackingAddress.ETH || tokenName === TokensWithBackingAddress.USDC || tokenName === TokensWithBackingAddress.USDT
```

Comment:
You could use _fall-through_ in the `switch` `case` to standardise without the need of the if check.